From: Ubuntu Developers <ubuntu-devel-discuss@lists.ubuntu.com>
Subject: Refresh move-test-requirements-out-of-runtime-requirements.patch for Hibiscus
Forwarded: no

Refresh existing pydot relocation from runtime to test requirements against current dependency ordering; retain test dependency.

--- a/requirements.txt
+++ b/requirements.txt
@@ -35,6 +35,3 @@
 
 # For lru caches and such
 cachetools>=2.0.0 # MIT License
-
-# For pydot output tests
-pydot>=1.2.4 # MIT License
--- a/test-requirements.txt
+++ b/test-requirements.txt
@@ -4,3 +4,4 @@
 testscenarios>=0.4 # Apache-2.0/BSD
 stestr>=2.0.0 # Apache-2.0
 pifpaf>=0.10.0 # Apache-2.0
+pydot>=1.2.4 # MIT License
